src/Backend/Session.vala

```diff
@@ -73,6 +78,30 @@ public class Mail.Backend.Session : Camel.Session {
                 critical (e.message);
             }
         });
+
+        manage_connection.begin ();
+    }
+
+    public async void manage_connection () {
```

Comment:
I'm not sure about this. It seems like we'd want to be able to manage the online state per account eventually

Reply:
I think (maybe I am missing something) it's currently not possible without some major development effort to put individual accounts to offline mode, because certain methods (e.g. `Camel.Store.get_folder_info`) automatically try to connect the particular account to the internet if the `Session` is online and I didn't find a way to prevent that (once again I am not really sure, maybe I am missing something). Nevertheless I now moved the method for connecting the individual accounts into the `Mail.Backend.Account` class and added a `local_only` variable to prevent a connection. But for the reason explained above even if set to true this currently doesn't really do anything (So should I remove it?).